packages/Webkul/Shop/src/Resources/assets/css/app.css

```diff
@@ -327,45 +327,49 @@
     }
 
     .wrapper-404 {
-			aspect-ratio: 16/9;
-			position: relative;
-			font-size: 394px;
-			perspective: 4.5em;
-			overflow: clip;
-			place-self: stretch;
-			top: 50%;
-			transform: translateY(-62%);
-		}
-
-		.glow-404,
-		.glow-shadow-404 {
-			position: absolute;
-			inset: 0;
-			display: grid;
-			place-content: center;
-			font-family: 'DM Serif Display';
-			overflow: hidden;
-			letter-spacing: 24px;
-		}
-
-		.glow-404 {
-			background-clip: text;
-			-webkit-background-clip: text;
-			color: #060C3B;
-		}
-
-		.glow-shadow-404 {
-			color: #E6E7EB;
-			mix-blend-mode: darken;
-			transform: translateY(15.5%) rotateX(243deg) scaleY(-1) translateY(-23%) scaleY(1.35) translateX(7%) skewX(-45deg);
-		}
-
-		canvas[resize] {
-			width: 100%;
-			height: 100vh;
-		}
-
-		html {
-			margin-top: 0px !important;
-		}
+        aspect-ratio: 16/9;
```

Comment:
Instead of hardcoded css, we should use tail @apply directive to apply properties